app/views/devise/registrations/new.html.haml

```diff
@@ -33,4 +33,4 @@
             = 'or continue with'
         = render partial: 'cenit/open_id/form', locals: { redirect_uri: "#{Cenit.homepage}#{session_path(resource_name)}", providers: providers }
 
-      = render partial: 'devise/shared/sing_in'
\ No newline at end of file
+      = render partial: 'devise/sign_in'
```

Comment:
I think the new path should be `devise/shared/sign_in` right?

Reply:
Yes, it's that way, I already corrected it